### DISCUSSION TITLE
替代高dpi缩放后游戏窗口显示不正常

### DISCUSSION THREAD

**Discussion by deadblue0910** [Blinue/Magpie#608] [Q&A] [answered]
系统dpi改为100%后是正常的，有没有让程序以dpi100%的情况下启动的方法？
![Snipaste_2023-05-16_00-19-16](https://github.com/Blinue/Magpie/assets/48860218/d01ea4c7-ddf5-4ae0-ba4a-6fdb5424a460)
![Snipaste_2023-05-16_00-24-40](https://github.com/Blinue/Magpie/assets/48860218/c5a51f45-dfcd-4fb7-99c1-051358e64f7a)

**Comment by Blinue** [comment]
“显示不正常”是什么意思？这个游戏有没有启用兼容模式？

**Reply by deadblue0910** [reply]
显示器125%缩放，兼容模式启用了，可能是老游戏系统的原因，勾选替代高dpi缩放时候的文本框第三行中间以下的内容和画面不显示，菜单栏的终了跑到第二行了，跟magpie没有关系，不知道有没有解决方法

**Reply by Blinue** [reply, marked as answer]
试试关闭高DPI缩放替代，使用GDI捕获

**Reply by deadblue0910** [reply]
有效，感谢大佬
